main.go

```diff
@@ -6,7 +6,7 @@ import (
 	"path"
 	"strconv"
 
-	"github.com/codegangsta/cli"
+	"github.com/docker/machine/cli"
```

Comment:
Any particularly compelling reason you're vendoring `codegangsta` in such a strange way?  Is there some change you wanted to make that the upstream project isn't keen on?